src/autosuggest/styles.scss

```diff
@@ -21,6 +21,10 @@
   flex-shrink: 0;
 }
 
+.dropdown-content {
+  display: contents;
```

Comment:
Looks like this property is not safari-friendly: https://caniuse.com/css-display-contents

Could you check if anything explodes there?

Reply:
Yes, I checked it in Safari already with VO and found no regression.

Reply:
What versions did you check? We support down to Safari v13
https://github.com/cloudscape-design/components/blob/d4e70a6f3fa929b8c620f7f4b3ebd805277c2c3c/package.json#L156

We can of course merge as is and make our screenshot tests fail, but we can also validate this ahead of time

Reply:
Just checked Safari 12.1.